matRad_calcDoseInit.m

```diff
@@ -160,13 +165,17 @@
 % Convert CT subscripts to coarse linear indices.
 [yCoordsV_voxDoseGrid, xCoordsV_voxDoseGrid, zCoordsV_voxDoseGrid] = ind2sub(dij.doseGrid.dimensions,VdoseGrid);
 
-% load base data% load machine file
-fileName = [pln.radiationMode '_' pln.machine];
-try
-   load([fileparts(mfilename('fullpath')) filesep 'basedata' filesep fileName '.mat']);
-catch
-   matRad_cfg.dispError('Could not find the following machine file: %s\n',fileName); 
+if pln.withoutMachineFile == 0
```

Comment:
when you use the dummy file as suggested, you could just use this loading process as usual.